src/TestFramework/PhpUnit/Config/Builder/InitialConfigBuilder.php

```diff
@@ -210,6 +211,11 @@ private function addRandomTestsOrderAttributesIfNotSet(string $version, \DOMXPat
         }
     }
 
+    private function removeCacheResultFileAttribute(\DOMXPath $xPath): void
+    {
```

Comment:
Shouldn't this be part of XmlConfigurationHelper?

Reply:
@sanmai 

Thank you - give me a moment!